Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/css/tooltip.css

```diff
@@ -23,31 +23,34 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
  
-.tooltip {
+ .tooltip {
     z-index: var(--topZIndex);
     position: absolute;
-    opacity: 80%;
```

Comment:
What is the thinking of dropping the opacity here?

Reply:
This is for changing the tooltip arrow, I made it use transform instead of border. so it will be a real arrow not a invisible square. The invisible part of that square will cover the dot, so the user cannot click on it.